commit aa8c77b4dcb51f105ef3d185e4ea57c25494f699
Author: Cole Robinson <crobinso@redhat.com>
Date:   Tue Aug 2 11:54:35 2011 -0400

    man: virt-image: Tweak option desc for --noreboot
    
    And regenerate all man pages

diff --git a/man/en/virt-clone.1 b/man/en/virt-clone.1
index 9df2fea..f3767f3 100644
--- a/man/en/virt-clone.1
+++ b/man/en/virt-clone.1
@@ -124,7 +124,7 @@
 .\" ========================================================================
 .\"
 .IX Title "VIRT-CLONE 1"
-.TH VIRT-CLONE 1 "2010-12-17" "" "Virtual Machine Install Tools"
+.TH VIRT-CLONE 1 "2011-07-28" "" "Virtual Machine Install Tools"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
@@ -232,6 +232,10 @@ guest \s-1XML\s0.
 .IP "\-\-print\-xml" 2
 .IX Item "--print-xml"
 Print the generated clone \s-1XML\s0 and exit without cloning.
+.IP "\-\-replace" 2
+.IX Item "--replace"
+Shutdown and remove any existing guest with the passed \f(CW\*(C`\-\-name\*(C'\fR before
+cloning the original guest.
 .IP "\-d, \-\-debug" 2
 .IX Item "-d, --debug"
 Print debugging information to the terminal when running the install process.
diff --git a/man/en/virt-convert.1 b/man/en/virt-convert.1
index bab188f..988ce23 100644
--- a/man/en/virt-convert.1
+++ b/man/en/virt-convert.1
@@ -124,7 +124,7 @@
 .\" ========================================================================
 .\"
 .IX Title "VIRT-CONVERT 1"
-.TH VIRT-CONVERT 1 "2011-07-18" "" "Virtual Machine Install Tools"
+.TH VIRT-CONVERT 1 "2011-07-28" "" "Virtual Machine Install Tools"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
diff --git a/man/en/virt-image.1 b/man/en/virt-image.1
index 520cc5e..a68d8d6 100644
--- a/man/en/virt-image.1
+++ b/man/en/virt-image.1
@@ -124,7 +124,7 @@
 .\" ========================================================================
 .\"
 .IX Title "VIRT-IMAGE 1"
-.TH VIRT-IMAGE 1 "2011-06-09" "" "Virtual Machine Install Tools"
+.TH VIRT-IMAGE 1 "2011-08-02" "" "Virtual Machine Install Tools"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
@@ -205,48 +205,49 @@ values.
 Parameters specific only to fully virtualized guest installs.
 .IP "\-\-noapic" 2
 .IX Item "--noapic"
-Disables \s-1APIC\s0 for fully virtualized guest (overrides value in \s-1XML\s0 descriptor)
+Force disable \s-1APIC\s0 for the guest.
 .IP "\-\-noacpi" 2
 .IX Item "--noacpi"
-Disables \s-1ACPI\s0 for fully virtualized guest (overrides value in \s-1XML\s0 descriptor)
+Force disable \s-1ACPI\s0 for the guest.
 .SS "Networking Configuration"
 .IX Subsection "Networking Configuration"
+.IP "\-w \s-1NETWORK\s0, \-\-network=NETWORK" 2
+.IX Item "-w NETWORK, --network=NETWORK"
+Connect the guest to the host network. See \fIvirt\-install\fR\|(1) for details
 .IP "\-m \s-1MAC\s0, \-\-mac=MAC" 2
 .IX Item "-m MAC, --mac=MAC"
 This is deprecated in favor of \f(CW\*(C`\-\-network ...,mac=MAC,...\*(C'\fR
 .IP "\-b \s-1BRIDGE\s0, \-\-bridge=BRIDGE" 2
 .IX Item "-b BRIDGE, --bridge=BRIDGE"
 This is deprecated in favor of \f(CW\*(C`\-\-network bridge=BRIDGE\*(C'\fR
-.IP "\-w \s-1NETWORK\s0, \-\-network=NETWORK" 2
-.IX Item "-w NETWORK, --network=NETWORK"
-Connect the guest to the host network. See \fIvirt\-install\fR\|(1) for details
 .SS "Graphics Configuration"
 .IX Subsection "Graphics Configuration"
-If no graphics option is specified, \f(CW\*(C`virt\-install\*(C'\fR will default to \-\-vnc
-if the \s-1DISPLAY\s0 environment variable is set, otherwise \-\-nographics is used.
+If no graphics option is specified, \f(CW\*(C`virt\-image\*(C'\fR will default to
+\&'\-\-graphics vnc' if the \s-1DISPLAY\s0 environment variable is set, otherwise
+\&'\-\-graphics none' is used.
+.IP "\-\-graphics \s-1TYPE\s0,opt1=arg1,opt2=arg2,..." 2
+.IX Item "--graphics TYPE,opt1=arg1,opt2=arg2,..."
+Specifies the graphical display configuration. This does not configure any
+virtual hardware, just how the guest's graphical display can be accessed.
+See \fIvirt\-install\fR\|(1) for details usage info.
 .IP "\-\-vnc" 2
 .IX Item "--vnc"
-Setup a virtual console in the guest and export it as a \s-1VNC\s0 server in
-the host. See \fIvirt\-install\fR\|(1) for details
+This option is deprecated in favor of \f(CW\*(C`\-\-graphics vnc,...\*(C'\fR
 .IP "\-\-vncport=VNCPORT" 2
 .IX Item "--vncport=VNCPORT"
-Request a permanent, statically assigned port number for the guest \s-1VNC\s0
-console. See \fIvirt\-install\fR\|(1) for details
+This option is deprecated in favor of \f(CW\*(C`\-\-graphics vnc,port=PORT,...\*(C'\fR
 .IP "\-\-vnclisten=VNCLISTEN" 2
 .IX Item "--vnclisten=VNCLISTEN"
-Address to listen on for \s-1VNC\s0 connections. See \fIvirt\-install\fR\|(1) for details.
+This option is deprecated in favor of \f(CW\*(C`\-\-graphics vnc,listen=LISTEN,...\*(C'\fR
 .IP "\-k \s-1KEYMAP\s0, \-\-keymap=KEYMAP" 2
 .IX Item "-k KEYMAP, --keymap=KEYMAP"
-Request that the virtual \s-1VNC\s0 console be configured to run with a non-English
-keyboard layout.
+This option is deprecated in favor of \f(CW\*(C`\-\-graphics vnc,keymap=KEYMAP,...\*(C'\fR
 .IP "\-\-sdl" 2
 .IX Item "--sdl"
-Setup a virtual console in the guest and display an \s-1SDL\s0 window in the
-host to render the output. See \fIvirt\-install\fR\|(1) for details
+This option is deprecated in favor of \f(CW\*(C`\-\-graphics sdl,...\*(C'\fR
 .IP "\-\-nographics" 2
 .IX Item "--nographics"
-Do not attach a graphical device to the guest. See
-\&\fIvirt\-install\fR\|(1) for details
+This option is deprecated in favor of \f(CW\*(C`\-\-graphics none\*(C'\fR
 .SS "Miscellaneous Options"
 .IX Subsection "Miscellaneous Options"
 .IP "\-p, \-\-print" 2
@@ -264,19 +265,14 @@ Shutdown and remove any existing guest with the passed \f(CW\*(C`\-\-name\*(C'\f
 installing from the image.
 .IP "\-\-noreboot" 2
 .IX Item "--noreboot"
-Prevent the domain from automatically rebooting after the install has
-completed.
+Prevent the domain automatatically booting after importing the image.
 .IP "\-\-skip\-checksum" 2
 .IX Item "--skip-checksum"
 Do not check disk images against checksums (if they are listed in the
 image xml).
 .IP "\-d, \-\-debug" 2
 .IX Item "-d, --debug"
-Print debugging information
-.IP "\-\-force" 2
-.IX Item "--force"
-Prevent interactive prompts. If the intended prompt was a yes/no prompt, always
-say yes. For any other prompts, the application will exit.
+Print debugging information.
 .SH "EXAMPLES"
 .IX Header "EXAMPLES"
 Create and start a guest called \f(CW\*(C`example\*(C'\fR with a \s-1VNC\s0 console from
diff --git a/man/en/virt-image.5 b/man/en/virt-image.5
index c8e84a5..b59124a 100644
--- a/man/en/virt-image.5
+++ b/man/en/virt-image.5
@@ -1,15 +1,7 @@
-.\" Automatically generated by Pod::Man 2.16 (Pod::Simple 3.07)
+.\" Automatically generated by Pod::Man 2.23 (Pod::Simple 3.14)
 .\"
 .\" Standard preamble:
 .\" ========================================================================
-.de Sh \" Subsection heading
-.br
-.if t .Sp
-.ne 5
-.PP
-\fB\\$1\fR
-.PP
-..
 .de Sp \" Vertical space (when we can't use .PP)
 .if t .sp .5v
 .if n .sp
@@ -53,7 +45,7 @@
 .el       .ds Aq '
 .\"
 .\" If the F register is turned on, we'll generate index entries on stderr for
-.\" titles (.TH), headers (.SH), subsections (.Sh), items (.Ip), and index
+.\" titles (.TH), headers (.SH), subsections (.SS), items (.Ip), and index
 .\" entries marked with X<> in POD.  Of course, you'll have to process the
 .\" output yourself in some meaningful fashion.
 .ie \nF \{\
@@ -132,7 +124,7 @@
 .\" ========================================================================
 .\"
 .IX Title "virt-image 5"
-.TH virt-image 5 "2009-12-01" "" "Virtual Machine Install Tools"
+.TH virt-image 5 "2011-07-28" "" "Virtual Machine Install Tools"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
@@ -205,7 +197,7 @@ recommended amount of memory in kB in the \fBmemory\fR element. It also
 indicates whether the virtual machine should have a network interface
 through the \fIinterface\fR element and whether the virtual machine has a
 graphical interface through the \fIgraphics\fR element.
-.Sh "\s-1BOOT\s0"
+.SS "\s-1BOOT\s0"
 .IX Subsection "BOOT"
 Each \fBboot\fR descriptor details how the virtual machine should be started
 on a certain hypervisor. The \fBtype\fR attribute of the \fBboot\fR element,
@@ -276,7 +268,7 @@ created, with the size given by the \fIsize\fR attribute.
 the \fIsize\fR attribute giving the size of the disk in \s-1MB\s0.
 .IP "\(bu" 4
 the \fIformat\fR attribute giving the format of the disk file. Currently, this
-can be one of: \f(CW\*(C`raw\*(C'\fR \f(CW\*(C`iso\*(C'\fR, \f(CW\*(C`qemu\*(C'\fR, \f(CW\*(C`qemu2\*(C'\fR, or \f(CW\*(C`vmdk\*(C'\fR.
+can be one of: \f(CW\*(C`raw\*(C'\fR \f(CW\*(C`iso\*(C'\fR, \f(CW\*(C`qcow\*(C'\fR, \f(CW\*(C`qcow2\*(C'\fR, or \f(CW\*(C`vmdk\*(C'\fR.
 .SH "EXAMPLE"
 .IX Header "EXAMPLE"
 The image descriptor below can be used to create a virtual machine running
diff --git a/man/en/virt-image.pod b/man/en/virt-image.pod
index 51eeec0..83a8afc 100644
--- a/man/en/virt-image.pod
+++ b/man/en/virt-image.pod
@@ -201,8 +201,7 @@ installing from the image.
 
 =item --noreboot
 
-Prevent the domain from automatically rebooting after the install has
-completed.
+Prevent the domain automatatically booting after importing the image.
 
 =item --skip-checksum
 
diff --git a/man/en/virt-install.1 b/man/en/virt-install.1
index 5a2e244..0b73192 100644
--- a/man/en/virt-install.1
+++ b/man/en/virt-install.1
@@ -124,7 +124,7 @@
 .\" ========================================================================
 .\"
 .IX Title "VIRT-INSTALL 1"
-.TH VIRT-INSTALL 1 "2011-07-26" "" "Virtual Machine Install Tools"
+.TH VIRT-INSTALL 1 "2011-07-28" "" "Virtual Machine Install Tools"
 .\" For nroff, turn off justification.  Always turn off hyphenation; it makes
 .\" way too many mistakes in technical documents.
 .if n .ad l
